assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
+
+(November 14, 2013) President Obama declares to press that he wasn’t good inform of the situation. The project team was not prepared for the failure.
+
+(November 30, 2013)  Enrollment statistics show that 137,000 individuals had signed up for plans using the site compared to the 227,000 that had enrolled through the 14 state run exchanges.
+
+Obama Administration set deadline of a working site for the “vast majority” of users. (December 1, 2013)
+
+
+### Failure causes & possible solutions.
+The failure was caused by a complex interplay of problems:
+
+##### Cause 1: Contractor Management.
+ 6. Contractor delays and performance issues were not always identified.
+ 7. A contractor incurred unauthorized costs that increased the cost of the contract.
+ 8. Contracting officers in all Government agencies did not have access to contractor past-performance evaluations when making contract awards.
+ 9. Critical deliverables and management decisions were not properly documented.
+ 
+> At this excerpt extracted from [source] we add the question: **where could it change?**. A stricter monitoring of the deliverables by the contractors (with small and testable deliverables), could have helped this project.
+
+
+##### Cause 2: Architecture, non functional requirements.
+ 1. The system was not designed to handle the massive influx of initial users. 
+ 2. Insufficient effort made to build a system that would meet the performance and availability needs of it's stakeholders. 
+ 3. Security solutions were seemingly slapped together in a shoddy manner instead of through the kind of systematic approach that is expected in a highquality software solution.
+ 4. Many thousands of pages of legal healthcare regulations did not translated seamlessly into functional requirements.
+ 5. The basic architecture was designed and built around the notion that the system would forward requests for quotes from insurance seekers to external vendors in real-time; however, this massive interconnectivity and the subsequent burden on the government servers caused the system to collapse.
+
+>However, all these points, extracted from [1] are a compilation of the problem's triggers , but **where could it change?**
+Besides all the disastrous organizational perspective, and scoping at the missing functional requirements, the elicitation, documentation and communication of those requirements probably could have helped this project. There are many structured techniques specifically designed to learn others about the system and what should it carry, some techniques such as 'planguage' or 'volere templates'.
+
+
+ 
+##### Cause 3: Architecture, Change Managment.
+The inability to handle a growing amount of work, its potential to be enlarged in order to accommodate that growth, or a limited space to accommodate iterations can be a very dangerous thing. In this case, three months before the deadline a requirement changed.
```

Comment:
"In this case, three months before the deadline a requirement changed."

"In this case, just a week before the deadline a requirement changed that had a major impact on the architectural constraints of the site."

Note it was also just a week before the deadline and not three months. My bad. I will also fix that in cause3.